### ISSUE TITLE
Clonning Main page content

### ISSUE THREAD

**Issue by bd0bd** [backdrop/backdrop-issues#2027]
I have deleted the Main page content block from Content area and placed it into Bottom area.
But the main content is outputed in both areas — Content and Bottom.

Is it a bug?

![aaa](https://cloud.githubusercontent.com/assets/13515302/17080112/9811bfc2-512c-11e6-9c51-ffb412049f67.png)

**Comment by stpaultim** [comment]
I was able to recreate this problem:

1) I created a node
2) I deleted the "main page content" block from the default layout. 
3) I added the "main page content" block to the "bottom" region
4) I navigated to the node page and found that the content was displayed twice, both in the "content" region and in the "bottom" region.

NOTE: Before I was able to recreate this, I tried to recreate it using a layout I had created for node/% - in this case things worked as expected. I was only able to recreate it when I used the default layout.
